pkg/security/probe/model.go

```diff
@@ -239,11 +239,15 @@ type ContainerEvent struct {
 }
 
 type KernelEvent struct {
-	Type      uint64 `field:"type"`
+	Type      uint64 `field:"type" handler:"ResolveType,string"`
 	Timestamp uint64 `field:"-"`
 	Retval    int64  `field:"retval"`
 }
 
+func (k *KernelEvent) ResolveType(resolvers *Resolvers) string {
```

Comment:
We should call it in the marshal function

Reply:
Fixed